src/components/store/SizeSelector/style.module.scss

```diff
@@ -0,0 +1,64 @@
+@use 'src/styles/vars.scss' as vars;
+$indicator-size: 30px;
+$component-roundness: 10px;
+
+.sizeSelector {
+  display: flex;
+  flex-direction: column;
+  gap: 0.75rem;
+  overflow: auto;
+
+  h4 {
+    color: var(--theme-text-on-background-1);
+    font-size: vars.$general-heading-3-4;
+    font-weight: bold;
+  }
+
+  .switch {
```

Comment:
![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/e2920b49-5bdb-4c81-8f5c-a6dd26f040e1)

you should change `width` to `min-width` for the labels (and add some horizontal padding) and add `flex-wrap: wrap;` to `.switch` so if there's not enough room, the options go onto the next line

![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/a95b2022-a04f-4176-a793-767dd8efc245)